SonarQube.Common/ConsoleLogger.cs

```diff
@@ -41,18 +45,28 @@ public ConsoleLogger(bool includeTimestamp)
         public void LogWarning(string message, params object[] args)
         {
             string finalMessage = this.GetFormattedMessage(Resources.Logger_WarningPrefix + message, args);
-            Console.WriteLine(finalMessage);
+
+            using (new ConsoleColorScope(WarningColor))
+            {
+                Console.WriteLine(finalMessage);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=csharpsquid%3AS2228)